.github/workflows/ci-complete.yml

```
@@ -0,0 +1,162 @@
+        if [ ! -f "backendiatech/package.json" ]; then
+          echo "❌ Error: package.json del backend no encontrado"
+          exit 1
+        fi
+        
+        echo "✅ Estructura del monorepo verificada correctamente"
+        
+    - name: Mostrar información del proyecto
+      run: |
+        echo "📊 Información del Monorepo IATECH:"
+        echo "Frontend: $(cd frontediatech && node -p 'require("./package.json").name + " v" + require("./package.json").version')"
+        echo "Backend: $(cd backendiatech && node -p 'require("./package.json").name + " v" + require("./package.json").version')"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)